release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -127,6 +127,13 @@ For more information, see xref:../networking/hardware_networks/about-sriov.adoc#
 You can update the Ingress Controller to switch between an AWS Classic Load Balancer (CLB) and an AWS Network Load Balancer (NLB) without deleting the Ingress Controller.
 
 For more information, see xref:../networking/configuring_ingress_cluster_traffic/configuring-ingress-cluster-traffic-aws.adoc[Configuring ingress cluster traffic on AWS].
+[id="ocp-4-12-ipv6-gratuitous-ARPs-default-SR-IOV"]
```

Comment:
@ShaunaDiaz, it looks like this line can move down one, and the space between this and the header (`====`) can be removed.

Thanks!